drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/custom_assessment_loader.rb

```diff
@@ -153,6 +155,15 @@ def create_form_processor_records(rows)
       ar_import(processor_model, records)
     end
 
+    def form_definition
+      @form_definition ||= Hmis::Form::Definition.where(identifier: form_definition_identifier).first_or_create! do |definition|
+        definition.title = form_definition_identifier.humanize
+        definition.status = 'draft'
+        definition.version = 0
+        definition.role = 'FORM_DEFINITION'
```

Comment:
this isn't valid, but we shouldn't hit this code

Reply:
Oh yeah, this should be 'CUSTOM_ASSESSMENT', thanks!